### PR TITLE
[ticket/14626] Event core.make_jumpbox_modify_tpl_ary

### PR DESCRIPTION
**Pull request by tas2580**
PHPBB3-14626

### CODE REVIEW

**Comment by rmcgirr83** [comment]
You have to unlink the tracker url from the PHPBB3-14626 I believe

**Comment by Crizz0** [comment]
You have to change the first commits message, too. Then the error will disappear.

File: phpBB/includes/functions_content.php [hunk 1]
```diff
@@ -227,7 +228,27 @@ function make_jumpbox($action, $forum_id = false, $select_all = false, $acl_list
...
+		 * @var	array	tpl_ary			Template data of the forum
+		 * @since 3.1.10-RC1
+		 */
+		$vars = array(
+			'row',
+			'tpl_ary',
+		);
+		extract($phpbb_dispatcher->trigger_event('core.make_jumpbox_modify_tpl_ary', compact($vars)));
+
+		foreach ($tpl_ary as $i => $ary)
```
Comment (marc1706) [inline phpBB/includes/functions_content.php]: This is not needed. You should instead use $template->assign_block_vars_array().

**Comment by marc1706** [comment]
Thanks for your pull request. I've taken the liberty to try and fix the commit message issue. See #4352

**Pull request closed without merging**